pycbc/inject/inject.py

```diff
@@ -403,15 +404,22 @@ def make_strain_from_inj_object(self, inj, delta_t, detector_name,
             f_l = inj.f_lower
         else:
             f_l = f_lower
-
-        name, phase_order = legacy_approximant_name(inj.waveform)
-
-        # compute the waveform time series
-        hp, hc = get_td_waveform(
-            inj, approximant=name, delta_t=delta_t,
-            phase_order=phase_order,
-            f_lower=f_l, distance=inj.distance,
-            **self.extra_args)
+	approx_name=inj.waveform #put the path to the ASCII file as approximant
+	if approx_name.endwith(('.txt', '.dat')):
```

Comment:
I don't think this method exists. That is why travis is failing the unittests.

Reply:
`endwith` -> `endswith` I think